### PR TITLE
docs(README.md): updated README.md

### PR DESCRIPTION
**Pull request by khubo**
library is pretty helpful but the docs were a bit confusing as it's assuming a generator sort of code already? found this [issue](https://github.com/mongodb-js/mongodb-topology-manager/issues/2).  probably, this PR should fix it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,17 +10,22 @@ It's very simple to create a single running MongoDB instance. All examples are u
...
+  // Purge the directory
+  await server.purge();
+  // Start process
+  await server.start();
+  // Stop the process
+  await server.stop();
+}
+
+// start the server
+init()
```
Comment (vkarpov15) [inline README.md]: Use semicolons consistently please

Comment (khubo) [reply README.md]: oops. sorry about the semicolons. havent used them in a while :sweat_smile: . will do if @mbroadst approves

Comment (mbroadst) [reply README.md]: @khubo LGTM! Please do as @vkarpov15 asks, but I'm on board with async-ifiying the documentation.

**Comment by khubo** [comment]
@mbroadst added semicolons wherever necessary :smile:

**Comment by mbroadst** [comment]
@khubo thanks!

**Pull request merged**